src/react-app.js

```diff
@@ -166,7 +166,6 @@ function renderReactApp({
             <Tab key={id} css={{display: 'flex', alignItems: 'center'}}>
               {React.createElement(getExtraIcon(index), {
                 size: 20,
-                color: theme.textLightest,
```

Comment:
Could you move this to just above line 122? https://github.com/kentcdodds/react-workshop-app/pull/24/files#diff-34eb4dcf11ddc16750d314d41a011363R122

Reply:
I can! But looks to me that it would alter how it looks.

`advanced-react-hooks` on my machine: 

<img width="422" alt="image" src="https://user-images.githubusercontent.com/5889721/95124928-2f5f7680-0722-11eb-8642-ef012907b429.png">

With the change requested above:

<img width="421" alt="image" src="https://user-images.githubusercontent.com/5889721/95124960-41d9b000-0722-11eb-9d86-d69e2d63d648.png">

`color: theme.textLightest,` was only being applied to the icon. Moving it just above line 122 would apply it to the whole tab.

All depends on what you want to achieve.

Reply:
Oooooooh, yeah that's not what I want 🙈 Thanks! I like the way it is without this change. Thanks!